lib/install/config/shared.js

```diff
@@ -28,7 +28,7 @@ const config = {
     rules: [
       { test: /\.coffee(\.erb)?$/, loader: 'coffee-loader' },
       {
-        test: /\.js(\.erb)?$/,
+        test: /\.jsx?(\.erb)?$/,
```

Comment:
I think we shouldn't have `jsx` here by default or people will think that this is going to work even when the `react` preset isn't set for Babel and it is not by default